src/scripts/main.js

```diff
@@ -1,3 +1,26 @@
 'use strict';
 
-// write your code here
+let total = 0;
+
+const calculateTotal = (num) => {
+  total += num;
+};
+
+const populations = document.querySelectorAll('.population');
+
+populations.forEach(el => {
+  const number = Number(el.textContent.split(',').join(''));
```

Comment:
Consider using a more descriptive variable name, like 'populationNumber', instead of 'number'.